src/main/java/model/User.java

```diff
@@ -1,16 +1,23 @@
 package model;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class User {
 
     private Long id;
     private String email;
     private String password;
     private String role;
+    private List<Order> orders;
```

Comment:
do we need this field here? and basket too?

Reply:
rebuilt project. fixed.